test/distributed/_composable/fsdp/_test_fully_shard_common.py

```diff
@@ -0,0 +1,20 @@
+# Owner(s): ["oncall: distributed"]
+
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class MLP(nn.Module):
```

Comment:
it feels to me that the `MLP` module is so popular in testing (dtensor also have [one](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/distributed/_tensor/common_dtensor.py#L53)), maybe as a follow up BE task we should consolidate these two and put it under `common_distributed`?

Reply:
Good point! (We will want a buffer on the MLP module though in a later PR 😄 .)